src/cohort-manager.js

```diff
@@ -0,0 +1,132 @@
+const Cohort = require('./cohort')
+const Student = require('./student')
+
+class CohortManager {
+  constructor() {
+    this.cohortArray = []
+    this.studentID = 1
+  }
+
+  addCohort(name, maxCapacity) {
+    if (name.length === 0) {
+      return false
+    }
+    for (let i = 0; i < this.cohortArray.length; i++) {
+      const cohort = this.cohortArray[i]
```

Comment:
I like how you create a variable COHORT and assign the value of the current iteration,
I understood very easily what you are doing there and it gave me an idea of how to do these things next time.

Reply:
Thanks, it certainly helps readability. At a glance, this.cohortArray[i] can be difficult to understand quickly